webserver/static/js/eval-jobs-viewer.js

```diff
@@ -237,10 +239,22 @@ var JobDetails = React.createClass({
                 </div>;
                 break;
             case JOB_STATUS_DONE:
-                status = <div className="alert alert-success">
-                    This evaluation job has been completed on {this.props.updated}.
-                    You can find results below.
-                </div>;
+		
+		switch (this.props.valid) {
+		case '1':
+                    status = <div className="alert alert-success">
+                        This evaluation job has been completed on {this.props.updated}.
+                        You can find results below.
+                    </div>;
+		    break;	
+		case '0':
+		    status = <div className="alert alert-success">
+                        This evaluation job has been completed on {this.props.updated}.
+                         You can find results below.But these results are outdated since the dataset was edited after the job was initiated.
```

Comment:
It would also be a good idea to show an indication on the job list:

![screen shot 2016-03-02 at 10 50 20 pm](https://cloud.githubusercontent.com/assets/19217/13476671/37aa0e38-e0c9-11e5-867f-366e02a91747.png)

perhaps instead of saying [Done], for outdated jobs we can say [Done, Outdated], in yellow